### PR TITLE
Feature/365 add tests for linked list

### PR DESCRIPTION
**Pull request by gorgos**
https://starbase.atlassian.net/browse/SP-365

Tests inspired by https://github.com/Modular-Network/ethereum-libraries/blob/master/LinkedListLib/truffle/test/TestLinkedListLib.sol

### CODE REVIEW

File: test/linkedListLib.test.js [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+    '0x4f2b82633c352389d109834e269e7d131b135b7d',
+    '0x8c83e1b66882b06e857806dec787361ea699ad22',
+];
+
+async function sizeShouldBe(linkedList, expectedSize) {
+    const size = await linkedList.sizeOf();
+    size.should.be.bignumber.equal(new BigNumber(size), `The size of the linked list should be ${expectedSize}!`);
+}
+
+contract.only('LinkedListMock', () => {
```
Comment (gustavoguimaraes) [inline test/linkedListLib.test.js]: Remove `.only` from tests

File: test/linkedListLib.test.js [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+require('chai')
+  .use(require('chai-bignumber')(BigNumber))
+  .should();
+
+const NULL = '0x0000000000000000000000000000000000000000';
+const HEAD = '0x0000000000000000000000000000000000000000';
+const PREV = false;
+const NEXT = true;
+
+const ADDRESSES = [
```
Comment (gustavoguimaraes) [inline test/linkedListLib.test.js]: These addresses can come from the second params of the `contract` block.

`contract('MyContract', [account1, account2, account3, account4, etc], => {...`

File: test/linkedListLib.test.js [hunk 3]
```diff
@@ -0,0 +1,255 @@
+const BigNumber = web3.BigNumber;
+const LinkedListMock = artifacts.require('LinkedListMock.sol');
+
+require('chai')
```
Comment (gustavoguimaraes) [inline test/linkedListLib.test.js]: This also can be changed to `const { should } = require('./helpers/utils');`

**Comment by gorgos** [comment]
>  You’re not authorized to merge this pull request.

:/

**Comment by gustavoguimaraes** [comment]
Try again

**Pull request merged**